.github/AL-Go-Settings.json

```diff
@@ -1,76 +1,76 @@
-{
-  "type": "PTE",
-  "templateUrl": "https://github.com/microsoft/AL-Go-PTE@preview",
-  "bcContainerHelperVersion": "preview",
-  "runs-on": "windows-latest",
-  "cacheImageName": "",
-  "UsePsSession": false,
-  "artifact": "https://bcinsider.azureedge.net/sandbox/24.0.15260.0/base",
-  "country": "base",
-  "useProjectDependencies": true,
-  "repoVersion": "24.0",
-  "cleanModePreprocessorSymbols": [
-    "CLEAN17",
-    "CLEAN18",
-    "CLEAN19",
-    "CLEAN20",
-    "CLEAN21",
-    "CLEAN22",
-    "CLEAN23",
-    "CLEAN24"
-  ],
-  "unusedALGoSystemFiles": [
-    "AddExistingAppOrTestApp.yaml",
-    "CreateApp.yaml",
-    "CreateOnlineDevelopmentEnvironment.yaml",
-    "CreatePerformanceTestApp.yaml",
-    "CreateRelease.yaml",
-    "CreateTestApp.yaml",
-    "Current.yaml",
-    "IncrementVersionNumber.yaml",
-    "NextMajor.yaml",
-    "NextMinor.yaml",
-    "PublishToEnvironment.yaml",
-    "Test Current.settings.json"
-  ],
-  "excludeEnvironments": [
-    "Official-Build"
-  ],
-  "buildModes": [
-    "Default",
-    "Clean"
-  ],
-  "CICDPushBranches": [
-    "main",
-    "release/*"
-  ],
-  "CICDPullRequestBranches": [
-    "main",
-    "release/*",
-    "features/*"
-  ],
-  "enableCodeCop": true,
-  "enableAppSourceCop": true,
-  "enablePerTenantExtensionCop": true,
-  "enableUICop": true,
-  "enableCodeAnalyzersOnTestApps":  true,
-  "rulesetFile": "..\\..\\..\\src\\rulesets\\ruleset.json",
-  "skipUpgrade": true,
-  "PartnerTelemetryConnectionString": "InstrumentationKey=403ba4d3-ad2b-4ca1-8602-b7746de4c048;IngestionEndpoint=https://swedencentral-0.in.applicationinsights.azure.com/",
-  "fullBuildPatterns": [
-    "build/*",
-    "src/rulesets/*",
-    ".github/workflows/PullRequestHandler.yaml",
-    ".github/workflows/_BuildALGoProject.yaml"
-  ],
-  "PullRequestTrigger": "pull_request",
-  "ALDoc": {
-    "maxReleases": 0,
-    "continuousDeployment": true,
-    "groupByProject": false,
-    "excludeProjects": [
-      "build_projects_System Application Modules"
-    ]
-  },
-  "templateSha": "8fe39e623ddffef215ec03093203d29ee63fe6d4"
-}
+{
+    "type":  "PTE",
+    "templateUrl":  "https://github.com/microsoft/AL-Go-PTE@preview",
+    "bcContainerHelperVersion":  "preview",
+    "runs-on":  "windows-latest",
+    "cacheImageName":  "",
+    "UsePsSession":  false,
+    "artifact":  "https://bcinsider.azureedge.net/sandbox/24.0.15328.0/base",
+    "country":  "base",
+    "useProjectDependencies":  true,
+    "repoVersion":  "24.0",
+    "cleanModePreprocessorSymbols":  [
+                                         "CLEAN17",

```

Comment:
is this new file formatting expected?

Reply:
Yeah, AL-Go changes the formatting every time.